source/release-notes.txt

```diff
@@ -20,6 +20,104 @@ Release Notes for |bi|
 .. expect and we don't break the links on the downloads page.
 .. (DOCS-9670 for context)
 
+.. _bi-2-10-0:
+
+|bi| 2.10.0
+-----------
+
+*Released Mar 15, 2019*
+
+Bug Fixes
+~~~~~~~~~
+
+- Fixes a bug where aggregate functions returned ``NULL`` in dual
+  queries with the ``DUAL`` keyword omitted.
+
+- Fixes a bug where some invalid ``DUAL`` queries did not return an
+  error.
+
+- Fix of a bug that prevented pushdown of some joins to derived tables
```

Comment:
for consistency, should this also be "fixes" instead of "fix of"?

Reply:
Good catch